.github/workflows/test-prs.yml

```diff
@@ -1,4 +1,7 @@
-# This workflow will run unit, e2e, and docs tests on PRs targeted against main, ui-svelte/main, ui-geo/main, and ui-react@v2 branches.
+# Description: This workflow runs unit + e2e tests on PRs targeting `main` and
+#              other protected branches listed below.
+#
+# Triggered by: internal and fork PRs targeting protected branches listed below.
```

Comment:
Cool! I was gonna say to add the tag name here as well, but it might be better to keep the code as the only source of truth

Reply:
Ooh you're right. I'll add that!

Reply:
c09748bef4c1073287c979575394d0786894c3f9